exchanges/gateio/gateio_test.go

```diff
@@ -3209,9 +2607,13 @@ func TestGetFuturesContractDetails(t *testing.T) {
 	require.NoError(t, err, "GetFuturesContractDetails must not error for DeliveryFutures")
 	assert.Equal(t, len(exp), len(c), "GetFuturesContractDetails should return same number of Delivery contracts as exist")
 
-	c, err = g.GetFuturesContractDetails(t.Context(), asset.Futures)
-	require.NoError(t, err, "GetFuturesContractDetails must not error for DeliveryFutures")
-	assert.NotEmpty(t, c, "GetFuturesContractDetails should return same number of Future contracts as exist")
+	c, err = g.GetFuturesContractDetails(t.Context(), asset.CoinMarginedFutures)
```

Comment:
Can be deduped

Reply:
Is this what you meant?

Fixed https://github.com/gbjk/gocryptotrader/commit/66d61c0dae

Reply:
Sorry I should of provided an example, but:

```
	for _, a := range []asset.Item{asset.CoinMarginedFutures, asset.USDTMarginedFutures} {
		c, err = g.GetFuturesContractDetails(t.Context(), a)
		require.NoError(t, err, "GetFuturesContractDetails must not error for %s", a)
		assert.NotEmpty(t, c, "GetFuturesContractDetails should return contracts for %s", a)
	}
```

Reply:
Yep, that's nearly exactly what I did.
Pushed it now, sorry for the delay.

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/21e20c26b4